### ISSUE TITLE
Providing a Translation Environment.

### ISSUE THREAD

**Issue by reindex-ot** [bluesky-social/social-app#2940]
**Is your feature request related to a problem? Please describe.**

Provision of a translation environment using Crowdin or Weblate.

**Describe the solution you'd like**

Provision of Crowdin or Weblate translation environment.

**Describe alternatives you've considered**

**Additional context**

**Comment by DearFox** [comment]
Yes, this is super necessary! It is much more convenient to work on a special website and be able to comment and suggest line translations, because at the moment, translating such a large project through Github is turning into a complete nightmare...
And this, in theory, will attract more people to translate the project, since not many translators use or want to understand github.

I think this will also help move all these discussions and issues to a more convenient and designated place.
#5137 #5227 #5256 #5344 #5217 #4433 [and so on...](https://github.com/bluesky-social/social-app/issues?page=1&q=translation)
Sorry for ping so many of you, I just think this proposal should get more attention.

**Comment by RomanPro100** [comment]
I second this! A project this big that offers this many languages MUST have a translation environment.

**Comment by surfdude29** [comment]
Yes, it would be great if translations for all localizations of Bluesky were managed on something like Crowdin.

In the meantime, some language teams have set up their own projects on Crowdin. They do their translation work there and then open a PR here on GitHub when they're ready. I know that at least the [Indonesian translators](https://crowdin.com/project/bluesky-id) and the [Ukrainian translators](https://crowdin.com/project/bsky-app-ua) do this.

**Comment by DearFox** [comment]
> Yes, it would be great if translations for all localizations of Bluesky were managed on something like Crowdin.
> 
> In the meantime, some language teams have set up their own projects on Crowdin. They do their translation work there and then open a PR here on GitHub when they're ready. I know that at least the [Indonesian translators](https://crowdin.com/project/bluesky-id) and the [Ukrainian translators](https://crowdin.com/project/bsky-app-ua) do this.

Yes, I definitely saw it. And this only suggests that there should be a single project for translation on such a site. For example, I could help with improving the translation into both Russian and Ukrainian. Like many other users.
Also, a link to the translation project should be available to all bsky users in the language selection settings menu. Something like "Help add or improve the translation"

**Comment by reoblauber** [comment]
Absolutely! I've been thinking about updating the Spanish localization for some time now but it not having a translation environment has put me off doing it. Having one would not only make contributing accessible to me, but also to many others.
Bluesky devs, please add one!

**Comment by reindex-ot** [comment]
When I spoke about this at X, the CEO responded, which I'll include here. he said that people might think he was there for sales.

![Screenshot_2024-09-21-18-16-00-055_com twitter android](https://github.com/user-attachments/assets/abdb68a7-0917-40e5-8028-795cee954986)

Post (Japanese text)
https://twitter.com/sergeydmytryshy/status/1837384848702353556

**Comment by reindex-ot** [comment]
@mozzius 
Could you please tell the boss of Bluesky that he wants to adopt Crowdin?

**Comment by dies** [comment]
Hello Bluesky team and community. Just wanted to say that the Crowdin team would be super happy to see Bluesky on Crowdin. We will do our best to help you set up and manage the crowd localisation initiative.

**Comment by brodieavoult** [comment]
I guess if Crowdin isn't an option, we could always just go with Weblate ¯\_(ツ)_/¯

**Comment by reindex-ot** [comment]
Bluesky support replied that there was a discussion going on and asked me to post there, so I did.
https://github.com/bluesky-social/atproto/discussions/2832

**Comment by surfdude29** [comment]
A hopeful update on this: @/pfrazee has [indicated](https://github.com/bluesky-social/social-app/pull/6186#issuecomment-2486664738) that he's hoping to get Crowdin set up sometime soon 🤞

> It's probably time to formalize this process and get better contact with our translators! ... I think we do need to get crowdin going so that we can formalize this. I'm sorry this won't be as fast as yall deserve. We're putting out a lot of fires! But I've got this high on my task list.

**Comment by Signez** [comment]
Just chiming in to say that I would welcome the addition of Crowdin: it would make our work a lot easier!

May I suggest that the current team of volunteer translators could have, if not a moderating role, at least a ‘trusted’ role on that platform?

Of course, I don't want to seem _too_ attached to the work we've already done in French with @surfdude29 for example, but it would be preferable to have the tools in place to ensure that the current glossary, typographical rules, etc. that we've taken the time and effort to follow through all current translations are followed by new contributors, including a way to recognise the role we had until now.

(but even if we don't, it will be a net positive anyway 👍)

**Comment by monty241** [comment]
As an alternative we are currently on: https://localazy.com/p/bsky

Localazy works fine for this type of app from what I can see. We are moving away with bleeding heart from it because our apps have approximately 50x times as many resources, for which it became cumbersome.

**Comment by softinterlingua** [comment]
> Hello Bluesky team and community. Just wanted to say that the Crowdin team would be super happy to see Bluesky on Crowdin. We will do our best to help you set up and manage the crowd localisation initiative.

This weekend i (we) set up a [project at Crowdin to translate Bluesky to interlingua](Following suggestions from another language request issues, [we enabled a Crowdin project ](crowdin.com/project/bluesky-in-interlingua)to translate Bluesky) (lang code:ia)

Unfortunately for us, Crowdin has the disadvantage to not to have interlingua as an officially supported language, unlike another conlangs or fictional languages like LOLCAT or Pirate English. Some PMs refused to include our language to their projects (i.e. LineageOS) because they do not know how to create a custom language or defined to accept only official languages. 
We worry it could happen with Bluesky, too.

**Comment by surfdude29** [comment]
Great news! Bluesky is now using Crowdin! Please see #7685 for all the details.
